lib/heuristics/dichotomous_approach.rb

```diff
@@ -111,30 +114,30 @@ def self.dichotomous_heuristic(service_vrp, job = nil, &block)
               block&.call(wrapper, avancement, total, msg, cost, time, sol)
             }
 
-            transfer_unused_vehicles(service_vrp, solution, sub_service_vrps) if index.zero? && solution
+            transfer_unused_vehicles(solution, sub_service_vrps) if index.zero? && solution
 
             solution
           }
           solution = solutions.reduce(&:+)
           log "dicho - level(#{service_vrp[:dicho_level]}) before remove_bad_skills unassigned rate " \
               "#{solution.unassigned_stops.size}/#{service_vrp[:vrp].services.size}: " \
-              "#{(solution.unassigned_stops.size.to_f / service_vrp[:vrp].services.size * 100).round(1)}%", level: :debug
+              "#{(solution.unassigned_stops.size.to_f / service_vrp[:vrp].services.size * 100).round(1)}%"
```

Comment:
Is it necessary to move these logs from :debug to :info ? The logs defined as :info are already barely readable due to their high quantity

Reply:
I asked the same question :thinking: I ended up activating them because
- without the logs, it is impossible to see what is wrong (and dicho split is random so not easy to reproduce)
- most of the logs I activated are either once or twice per job or per sub-problem -- so we won't have 100s/1000s of extra lines of log because of them.